.github/workflows/ci.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Arrange
+      - name: Cancel Previous Runs
```

Comment:
I think we need to do more config to allow the cancel action to work with forks? [^cancel-fork]

We should also restrict the permissions for the action as much as possible, so it only uses what it really needs, and no more. [^cancel-permissions]

[^cancel-fork]: https://github.com/styfle/cancel-workflow-action#advanced-pull-requests-from-forks

[^cancel-permissions]: https://github.com/styfle/cancel-workflow-action#advanced-token-permissions

Reply:
I don't think we need to worry about forks since those are run outside our repo/org.

I'll set the proper token permission tho.

Reply:
I've set the read-only permission as explained in their docs:

<img width="808" alt="imagen" src="https://user-images.githubusercontent.com/2677072/139936100-da55d7c7-c991-4eb1-8368-95a2673902d8.png">

And set the `actions: write` in the workflow yml file.

Reply:
Okay so the repository defaults to `read` permission, unless we elevate it manually in the workflow `.yml` files. That's a good security posture to have as a baseline.